setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php

```diff
@@ -123,4 +124,31 @@ public function validate(InputInterface $input)
 
         return $errors;
     }
+
+    /**
+     * Get admin user data
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return void
+     */
+    protected function setMissingValues(InputInterface &$input, OutputInterface $output)
+    {
+        if (!method_exists($input, 'promptForOption')) {
```

Comment:
Isn't that what this is? https://github.com/magento/magento2/pull/2690/files#diff-6c32687b440c3791df9e43cba7f888b3